gvr-360video/app/src/main/java/org/gearvrf/gvr360video/Minimal360Video.java

```diff
@@ -33,7 +33,7 @@
     /** Called when the activity is first created. */
     @Override
     public void onInit(GVRContext gvrContext) {
-        GVRScene scene = gvrContext.getNextMainScene();
+        GVRScene scene = gvrContext.getMainScene();
```

Comment:
@liaxim if the plan is to only allow a single scene in the future then this change is fine.